src/components/Books.js

```diff
@@ -1,28 +1,20 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import NewBook from './NewBook';
 import AddBook from './AddBook';
 
 function Books() {
+  const bookList = useSelector((state) => state.booksReducer);
+
   return (
     <>
-      <NewBook
-        title="Hunger Games"
-        author="Suzanne Collins"
-        status="64%"
-        chapter="chapter 17"
-      />
-      <NewBook
-        title="Hunger Games"
-        author="Suzanne Collins"
-        status="64%"
-        chapter="chapter 17"
-      />
-      <NewBook
-        title="Hunger Games"
-        author="Suzanne Collins"
-        status="64%"
-        chapter="chapter 17"
-      />
+      {bookList.length ? (
+        bookList.map((book) => <NewBook book={book} key={book.id} />)
+      ) : (
```

Comment:
- [ ] App not working please kindly fix it 
<img width="1371" alt="image" src="https://user-images.githubusercontent.com/37901469/201058367-a11fb736-6850-4ac7-a811-8b237fad43a8.png">